test/models/order_item_test.rb

```diff
@@ -1,7 +1,15 @@
 require 'test_helper'
 
 class OrderItemTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include CategoryItemsSetup
+  test 'items are asscociated wit an order' do
+    create_items
+    order = Order.new(current_status: 'completed')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.